src/components/projecteditor/panes.js

```diff
@@ -308,16 +308,10 @@ class Panes extends Component {
     render() {
         const panes = this.renderPanes();
 
-        const { isActionPanelShowing } = this.props;
-
         return (
             <div
-                key="panes"
```

Comment:
Is it 100% safe to remove this?

Reply:
Seem to work ok. It is very strange there was a "key" attribute, as keys are designed only for rendering lists/arrays. And it is the case here.

Reply:
Ok, good then :)